frontend/src/metabase/common/components/FilterPicker/BooleanFilterPicker/BooleanFilterPicker.tsx

```diff
@@ -49,7 +50,7 @@ export function BooleanFilterPicker({
   };
 
   return (
-    <Box data-testid="boolean-filter-picker">
+    <Box miw={MIN_WIDTH} maw={MAX_WIDTH} data-testid="boolean-filter-picker">
```

Comment:
Here and in other components - moved `MIN_WIDTH` from `FilterPicker` to each component.